README.md

```diff
@@ -1,16 +1,16 @@
 # How to use Open Virtual Networking with Kubernetes
 
-On Linux, the easiest way to get started is to use OVN daemonset and deployments.
+On Linux, the easiest way to get started is to use OVN DaemonSet and Deployments.
 
 ## Install Open vSwitch kernel modules on all hosts.
 
-Most Linux distributions come with openvswitch kernel module by default.  You
+Most Linux distributions come with OpenvSwitch kernel module by default.  You
```

Comment:
well, if you're going for correctness, it's "Open vSwitch" with a space

Reply:
done